packages/yoastseo/src/languageProcessing/researches/findKeyphraseInSEOTitle.js

```diff
@@ -60,7 +61,7 @@ const adjustPosition = function( title, position ) {
 	}
 
 	// Strip all function words from the beginning of the SEO title.
-	const titleBeforeKeyword = title.substr( 0, position );
+	const titleBeforeKeyword = title.substring( 0, position );
```

Comment:
Changed `. substr` to `.substring` since the former is deprecated. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr